packages/account/src/Components/api-token/api-token-card.tsx

```diff
@@ -3,38 +3,26 @@ import { Field, FieldProps } from 'formik';
 import { CompositeCheckbox } from '@deriv/components';
 
 type TApiTokenCard = {
-    description: string;
+    description: JSX.Element;
```

Comment:
```suggestion
    description: React.ReactNode;
```

Reply:
description can't be null or undefined ...I think JSX.Element would be a better choice?